source/docs/a3.hello_world.md

```diff
@@ -8,14 +8,14 @@ fn main() {
 }
 ```
 
-`fn` means function. main function is the beginning of every Rust program.  
-`println!` prints text to the console and its *!* indicate that it’s a [macro](https://doc.rust-lang.org/book/first-edition/macros.html) instead of a function.
+`fn` means function. As in Clang, the `main` function is the beginning of every Rust program.  
```

Comment:
> As in Clang

Let's keep this generic to all audience by not targeting only systems programmers :)